commons/src/main/java/org/infinispan/multimap/api/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * MultimapCache API.
+ *
+ * @public
+ */
+package org.infinispan.multimap.api;
```

Comment:
Do we need the `api` package? Why `o.i.multimap` is not sufficient?
Also since this is in commons, it should rather be `org.infinispan.commons.multimap[.api]`. I've seen some problems if multiple modules contain the same package.

Reply:
I'm following how it is done in Counters. Anyway, for me multimap and after api ... I would prefer org.infinispan.api.multimap, org.infinispan.api.counters, org.infinispan.api.locks etc ...

Reply:
Yes, I'd prefer that too